src/stores/OnThisPageSectionsStoreBase.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * This source file is part of the Swift.org open source project
+ *
+ * Copyright (c) 2021 Apple Inc. and the Swift project authors
+ * Licensed under Apache License v2.0 with Runtime Library Exception
+ *
+ * See https://swift.org/LICENSE.txt for license information
+ * See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+export default {
+  state: {
+    onThisPageSections: [],
+    currentPageSection: null,
```

Comment:
Do you need to keep track of the entire current section object or would just the anchor be enough?

Reply:
I think we can keep only the anchor yes. It would be a bit more annoying to find the metadata for the current item, but not a big deal.

Reply:
Ok, just curious

Reply:
I refactored that, so gonna resolve this convo.